lib/sys.js

```diff
@@ -24,6 +24,8 @@
 // The sys module was renamed to 'util'. This shim remains to keep old programs
```

Comment:
Is this still the case? IIUC the case for keeping the module now is supply chain security, so maybe this comment should also be updated?

Reply:
It is still the case.